app/src/main/java/com/google/samples/apps/sunflower/GardenActivity.kt

```diff
@@ -50,6 +50,7 @@ class GardenActivity : AppCompatActivity() {
         drawerLayout = findViewById(R.id.drawer_layout)
         drawerToggle = ActionBarDrawerToggle(
                 this, drawerLayout, R.string.drawer_open, R.string.drawer_close)
+        drawerToggle.isDrawerSlideAnimationEnabled = false
```

Comment:
there are no reason to enable it. 

1) the specs did not mention about animation
2) it would have hidden by the drawer anyway
3) historically, it was used for a short time until the official support lib v21 released. which suggest the use of `Toolbar` with `setNavigationIcon` and `setNavigationOnClickListener` which is more simplified than using drawer toggle